tests/test_future_fingerprint.py

```
@@ -0,0 +1,36 @@
+    hashes, version = decompress_fingerprint(compressed)
+    assert hashes == TEST_2_FP_RAW
+    assert version == 1
+
+
+def test_decompress_invalid_data() -> None:
+    with pytest.raises(ValueError, match="Failed to decompress fingerprint"):
+        decompress_fingerprint(b"invalid data")
+
+    with pytest.raises(ValueError, match="Invalid fingerprint magic"):
+        decompress_fingerprint(
+            zstd.compress(struct.pack("<HBB", MAGIC + 99, FORMAT, 1))
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)